index.html

```diff
@@ -15,101 +16,72 @@
-                    <li>Установка дренажного насоса</li>
-                    <li>Прокладка канализационных труб</li>
-                    <li>Течёт под раковиной</li>
-                </ul>
+            <div class="card">
+                <img class="icon" src="./img/repair.png" alt="repair">
+                <h3 class="title"> З нами ваші ремонтні проблеми стають нашими. Звертайтеся до нас, і ми
+                    допоможемо вам зробити ваше житло ще затишнішим та комфортнішим!.</h3>
             </div>
-            <div class="container-with-text">
-                <p class="text">ЭЛEКТРИКА</p>
-                <ul class="list">
-                    <li>Монтаж электропроводки</li>
-                    <li>Замена электропроводки</li>
-                    <li>Замена автоматов</li>
-                    <li>Замена розеток</li>
-                    <li>Ремонт электроплиты</li>
-                    <li>Подключение люстры, бра</li>
-                </ul>
+            <div class="card">
+                <img class="icon" src="./img/safety.png" alt="safety">
+                <h3 class="title">Безпека - наш пріоритет. Ми пропонуємо широкий спектр послуг та рішень
+                    для забезпечення безпеки вас, вашої родини та вашого бізнесу. </h3>
             </div>
-            <div class="container-with-text">
-                <p class="text">БЫТОВАЯ ТЕХНИКА</p>
-                <ul class="list">
-                    <li>Ремонт стиральных/сушильных машин</li>
-                    <li>Ремонт холоильников</li>
-                    <li>Ремонт посудомоечных машин</li>
-                    <li>Монтаж жалюзи</li>
-                    <li>Установка зеркал</li>
-                    <li>Установка карнизов</li>
-                </ul>
+            <div class="card">
+                <img class="icon" src="./img/wiring.png" alt="wiring">
+                <h3 class="title">Потужний, надійний та безпечний - це те, що ми гарантуємо вам з нашими
+                    послугами з ремонту проводки. Довірте нам свою безпеку та спокійно насолоджуйтеся життям без
+                    обмежень!".</h3>
             </div>
-            <div class="container-with-text">
-                <p class="text">Другие работы</p>
-                <ul class="list">
-                    <li>Сборка мебели</li>
-                    <li>Ремонт мебели</li>
-                    <li></li>
-                </ul>
+            <div class="card">
+                <img class="icon" src="./img/installation.png" alt="installation">
+                <h3 class="title">Професійна інсталяція, без зайвих клопотів. Наша команда експертів
+                    забезпечить швидке та надійне встановлення вашого обладнання, системи чи пристроїв.</h3>
             </div>
         </div>
-    </div>
-    <div class="footer">
-        <img class="footer-logo" src="./img/plumber-footer.webp" alt="logo">
+    </main>
+    <footer>
+        <img class="footer-logo" src="./img/logo.webp" alt="logo">
```

Comment:
Great stuff overall! Let's simplify, refactor and correct several things.
1. Since you use nested class names in sass, it doesn't make sense to add `footer` prefix or postfix top classes which are already nested in footer. So, rename `footer-logo` with just `logo`, `footer-container` with `contacts`, `tel-footer` with `tel`, `footer-text` with `title`.